packages/shared/lib/core/layer-2/utils/getEstimatedGasForTransferFromTransactionDetails.ts

```diff
@@ -32,8 +29,12 @@ export async function getEstimatedGasForTransferFromTransactionDetails(
             const gasBurned = BigInteger(data.gasBurned as string).toJSNumber()
             const gasFeeCharged = BigInteger(data.gasFeeCharged as string).toJSNumber()
 
-            return { gasBurned, gasFeeCharged }
+            if (gasBurned && gasFeeCharged) {
+                return { gasBurned, gasFeeCharged }
+            }
         }
+
+        throw new Error(localize('error.layer2.estimatedGas'))
```

Comment:
If we are going to throw an error, I think it would be a good idea to handle it showing a notification
If not, the application crashes
![Captura de pantalla de 2023-11-16 15-13-58](https://github.com/iotaledger/firefly/assets/100352899/7f93cc89-6f35-47f8-a5a7-22e53136dd0c)

Reply:
It doesnt crash so dramatically for me 🤔 The error bubbles and its handled later on

Reply:
ok found it, it was a promise reject that was crashing everything, should be fixed now